deebot_client/commands/xml/pos.py

```diff
@@ -0,0 +1,53 @@
+"""Error commands."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from deebot_client.events import Position, PositionType, PositionsEvent
+from deebot_client.message import HandlingResult
+
+from .common import XmlCommandWithMessageHandling
+
+if TYPE_CHECKING:
+    from xml.etree.ElementTree import Element
+
+    from deebot_client.event_bus import EventBus
+
+
+class GetPos(XmlCommandWithMessageHandling):
+    """Get pos command."""
+
+    name = "GetPos"
+
+
+    @classmethod
+    def _handle_xml(cls, event_bus: EventBus, xml: Element) -> HandlingResult:
+        """Handle xml message and notify the correct event subscribers.
+
+        :return: A message response
+        """
+        positions = []
+
+        ret = str(ret) if (ret := xml.attrib["ret"]) else ""
+        if ret != "ok":
+            return HandlingResult.analyse()
+
+        # TODO: it is not really int:
+        # <ctl ret='ok' t='p' p='-5,676' a='92' valid='1'/>,
+        # <ctl ret='ok' t='p' p='2951,-463' a='-3' valid='1'/>",
+        p = int(float(p.replace(",","."))) if (p := xml.attrib["p"]) else 0
+        a = int(float(a.replace(",","."))) if (a := xml.attrib["a"]) else 0
+
+        positions.append(
+            Position(
+                type=PositionType.DEEBOT,
+                x=p,
+                y=a,
```

Comment:
From the examples, it looks like `p` is the point and `a` is the angle
So I think this is not correct. Can you please verify it